tools/src/main/java/com/example/core/broker/testkit/SubscribeToBuffer.java

```diff
@@ -0,0 +1,47 @@
+package com.example.core.broker.testkit;
+
+import akka.actor.ActorRef;
+
+public class SubscribeToBuffer {
```

Comment:
Thanks, will rewrite.
I was aware of the scala convention but forgot to check if there was one for java too. When porting to core I rewrote almost all in scala but we'll keep this here until bumping online-auction to 1.3 anyway so it'd better be convention-compliant.